src/main/java/com/mraof/minestuck/event/ServerEventHandler.java

```diff
@@ -166,6 +177,72 @@ public static void onEntityDamage(LivingHurtEvent event)
+						injuredPlayer.setHealth(injuredPlayer.getHealth() + 3);
+						injuredPlayer.addPotionEffect(new EffectInstance(Effects.REGENERATION, 450, 0));
+						injuredPlayer.addPotionEffect(new EffectInstance(Effects.ABSORPTION, 100, 0));
+						handItem.damageItem(400, injuredPlayer, playerEntity -> playerEntity.sendBreakAnimation(Hand.MAIN_HAND));
+					}
+					if(title.getHeroAspect() != EnumAspect.DOOM && injuredPlayer.getHealth() <= 2.0F && injuredPlayer.getRNG().nextFloat() <= .02)
+					{
+						injuredPlayer.world.playSound(null, injuredPlayer.getPosX(), injuredPlayer.getPosY(), injuredPlayer.getPosZ(), SoundEvents.ITEM_TOTEM_USE, SoundCategory.PLAYERS, 1.0F, 1.4F);
+						injuredPlayer.setHealth(injuredPlayer.getHealth() + 3);
+						injuredPlayer.addPotionEffect(new EffectInstance(Effects.REGENERATION, 450, 0));
+						handItem.damageItem(1000, injuredPlayer, playerEntity -> playerEntity.sendBreakAnimation(Hand.MAIN_HAND));
+					}
+				}
+				
+				if(handItem.getItem() == MSItems.CRUEL_FATE_CRUCIBLE){
+					if(title.getHeroAspect() == EnumAspect.DOOM && injuredPlayer.getHealth() <= 8.0F && injuredPlayer.getRNG().nextFloat() <= .25)
+					{
+						AxisAlignedBB axisalignedbb = injuredPlayer.getBoundingBox().grow(4.0D, 2.0D, 4.0D);
+						List<LivingEntity> list = injuredPlayer.world.getEntitiesWithinAABB(LivingEntity.class, axisalignedbb);
+						if (!list.isEmpty()) {
+							for(LivingEntity livingentity : list) {
+								double d0 = injuredPlayer.getDistanceSq(livingentity);
+								if (d0 < 16.0D) {
+									injuredPlayer.world.playSound(null, injuredPlayer.getPosX(), injuredPlayer.getPosY(), injuredPlayer.getPosZ(), SoundEvents.ENTITY_WITHER_HURT, SoundCategory.PLAYERS, 0.5F, 1.6F);
+									livingentity.addPotionEffect(new EffectInstance(Effects.INSTANT_DAMAGE, 1, 0));
+									injuredPlayer.removePotionEffect(Effects.INSTANT_DAMAGE);
+									handItem.damageItem(1, injuredPlayer, playerEntity -> playerEntity.sendBreakAnimation(Hand.MAIN_HAND));
+								}
+							}
+						}
+					}
+					if(title.getHeroAspect() != EnumAspect.DOOM && injuredPlayer.getHealth() <= 4.0F && injuredPlayer.getRNG().nextFloat() <= .10)
+					{
+						AxisAlignedBB axisalignedbb = injuredPlayer.getBoundingBox().grow(6.0D, 4.0D, 6.0D);
+						List<LivingEntity> list = injuredPlayer.world.getEntitiesWithinAABB(LivingEntity.class, axisalignedbb);
+						if (!list.isEmpty()) {
+							for(LivingEntity livingentity : list) {
+								double d0 = injuredPlayer.getDistanceSq(livingentity);
+								if (d0 < 16.0D) {
+									injuredPlayer.world.playSound(null, injuredPlayer.getPosX(), injuredPlayer.getPosY(), injuredPlayer.getPosZ(), SoundEvents.ENTITY_WITHER_HURT, SoundCategory.PLAYERS, 0.5F, 1.6F);
+									livingentity.addPotionEffect(new EffectInstance(Effects.INSTANT_DAMAGE, 1, 0));
+									injuredPlayer.removePotionEffect(Effects.INSTANT_DAMAGE);
```

Comment:
Why are you removing the potion effect from the injured player? If it is to avoid/*fix* the player from not getting the effect, I'd suggest instead checking livingentity beforehand such that it is not the injuredPlayer.

Reply:
Had trouble doing what you are suggesting originally, so kept this format. I had thought that it checks through individual LivingEntities as it moves through the list. I believe I originally tried to do something like this as an if statement: "d0 < 16.0 && livingentity != injuredPlayer" but whatever code I had put for that stopped it from activating at any point

Reply:
Sounds like something worth debugging then

Reply:
I fixed it, but not in the way I originally hoped and intended to!

Reply:
I have now fixed it, and in the way I intended to